### DISCUSSION TITLE
✅ 🔗 Link styling controls for Text Editor widget (Duplicate of #10855)

### DISCUSSION THREAD

**Discussion by hein-obox** [elementor/elementor#25430] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When I add hyperlinks inside my text editor widget, I am not able to update the styling of my hyperlinks on a non-global level.

### Describe the solution you'd like.

I would like the style tab of the text editor widget to have styling options for hyperlinks.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there Hein! 🙂 

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.28.0! 🥳 

Now you can change the color of links inside the Text Editor widget! 💪  

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

**Comment by hein-obox** [comment]
@nicholaszein That's great news!!!

**Comment by hein-obox** [comment]
@rami-elementor Thank you!
